bgpd/bgp_route.c

```diff
@@ -3059,6 +3079,22 @@ int bgp_update(struct peer *peer, struct prefix *p, uint32_t addpath_id,
 	else
 		has_valid_label = bgp_is_valid_label(label);
 
+	/* The flag BGP_NODE_FIB_INSTALL_PENDING is for the following
+	 * condition :
+	 * Suppress fib is enabled
+	 * BGP_OPT_NO_FIB is not enabled
+	 * BGP process is connected to RIB
+	 * Route type is BGP_ROUTE_NORMAL (peer learnt routes)
+	 * Route is being installed first time (BGP_NODE_FIB_INSTALLED not set)
+	 */
+	if ((bgp_zebra_num_connects() == 0) || bgp_option_check(BGP_OPT_NO_FIB))
```

Comment:
Do we even care about the number of connects here?  If we are going to bring up a connection to zebra and we let routes out *before* they can be passed down.  Do we ever come back and refigure out what to withdraw?